sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -1,477 +1,225 @@
-      CASE
-        WHEN app_opened_as_default_browser > 0
-          THEN client_id
-      END
-    ) AS app_opened_as_default_browser_users,
-    SUM(app_opened_as_default_browser) AS app_opened_as_default_browser,
-    -- settings_menu_set_as_default_browser_pressed
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN settings_menu_set_as_default_browser_pressed > 0
-          THEN client_id
-      END
-    ) AS settings_menu_set_as_default_browser_pressed_users,
-    SUM(
-      settings_menu_set_as_default_browser_pressed
-    ) AS settings_menu_set_as_default_browser_pressed,
-    /*Notification*/
-    --preferences_sync_notifs
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN preferences_sync_notifs > 0
-          THEN client_id
-      END
-    ) AS preferences_sync_notifs_users,
-    SUM(preferences_sync_notifs) AS preferences_sync_notifs,
-    -- preferences_tips_and_features_notifs
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN preferences_tips_and_features_notifs > 0
-          THEN client_id
-      END
-    ) AS preferences_tips_and_features_notifs_users,
-    SUM(preferences_tips_and_features_notifs) AS preferences_tips_and_features_notifs,
-    /*Customize Home*/
-    --preferences_jump_back_in
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN preferences_jump_back_in > 0
-          THEN client_id
-      END
-    ) AS preferences_jump_back_in_users,
-    SUM(preferences_jump_back_in) AS preferences_jump_back_in,
-    -- Preferences Recently Visited
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN preferences_recently_visited > 0
-          THEN client_id
-      END
-    ) AS preferences_recently_visited_users,
-    SUM(preferences_recently_visited) AS preferences_recently_visited,
-    -- Preferences Recently Saved
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN preferences_recently_saved > 0
-          THEN client_id
-      END
-    ) AS preferences_recently_saved_users,
-    SUM(preferences_recently_saved) AS preferences_recently_saved,
-    -- Preferences Pocket
-    COUNT(DISTINCT CASE WHEN preferences_pocket > 0 THEN client_id END) AS preferences_pocket_users,
-    SUM(preferences_pocket) AS preferences_pocket,
-    -- App Menu Customize Homepage
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN app_menu_customize_homepage > 0
-          THEN client_id
-      END
-    ) AS app_menu_customize_homepage_users,
-    SUM(app_menu_customize_homepage) AS app_menu_customize_homepage,
-    -- Firefox Home Page Customize Homepage Button
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN firefox_home_page_customize_homepage_button > 0
-          THEN client_id
-      END
-    ) AS firefox_home_page_customize_homepage_button_users,
-    SUM(firefox_home_page_customize_homepage_button) AS firefox_home_page_customize_homepage_button
-  FROM
-    product_features
-  GROUP BY
-    submission_date
+    -- In rare cases we can have an end_time that is earlier than the start_time, we made the decision
```

Comment:
Is there any discussion or ticket we could link to where this decision was made and why? Alternatively, maybe we could expand on this comment to provide more context?